cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -963,6 +963,58 @@ public void testTableSegmentRecoveryCommand() throws Exception {
         Assert.assertArrayEquals(Files.readAllBytes(Paths.get(pravegaStorageDir.toString(), listOfFiles.get(0).getName())), serializedEntries.getCopy());
     }
 
+    @Test
+    public void testTableSegmentRecoveryCommandVersioned() throws Exception {
+        // set pravega properties for the test
+        STATE.set(new AdminCommandState());
+        Properties pravegaProperties = new Properties();
+        pravegaProperties.setProperty("pravegaservice.container.count", "1");
+        pravegaProperties.setProperty("pravegaservice.storage.impl.name", "FILESYSTEM");
+        pravegaProperties.setProperty("pravegaservice.storage.layout", "CHUNKED_STORAGE");
+        pravegaProperties.setProperty("filesystem.root", this.baseDir.getAbsolutePath());
+        pravegaProperties.setProperty("bookkeeper.ledger.path", "/pravega/bookkeeper/ledgers0");
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+
+        // Create the data to test.
+        File testDataDir = Files.createTempDirectory("test-data-table-segment-recovery").toFile().getAbsoluteFile();
+        File pravegaStorageDir = Files.createTempDirectory("table-segment-recovery-command").toFile().getAbsoluteFile();
+
+        List<TableEntry> tableSegmentVersionedPuts = List.of(
+                TableEntry.versioned(new ByteArraySegment("kv1".getBytes()), new ByteArraySegment("vv1".getBytes()), 1L ),
```

Comment:
Nit: extra space before the closing parenthesis.

Reply:
fixed